dex/networks/bch/params.go

```diff
@@ -0,0 +1,51 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org
+
+package bch
+
+import (
+	"decred.org/dcrdex/dex/networks/btc"
+	"github.com/btcsuite/btcd/chaincfg"
+	bchchaincfg "github.com/gcash/bchd/chaincfg"
+)
+
+var (
+	// MainNetParams are the clone parameters for mainnet.
+	MainNetParams = btc.ReadCloneParams(&btc.CloneParams{
+		PubKeyHashAddrID: bchchaincfg.MainNetParams.LegacyPubKeyHashAddrID,
+		ScriptHashAddrID: bchchaincfg.MainNetParams.LegacyScriptHashAddrID,
+		Bech32HRPSegwit:  "bitcoincash",
+		CoinbaseMaturity: 100,
+		// BitcoinCash didn't change any of these net identifiers, so these are
+		// all spoofed. The only place we currently use Net is in
+		// btcd/chaincfg.Register, where it is checked to prevent duplicate
+		// registration, so our only requirement is that it is unique. This one
+		// was just generated with a prng.
+		Net: 0xe8f3e1e3,
+	})
+	// TestNet3Params are the clone parameters for testnet.
+	TestNet3Params = btc.ReadCloneParams(&btc.CloneParams{
+		PubKeyHashAddrID: bchchaincfg.TestNet3Params.LegacyPubKeyHashAddrID,
+		ScriptHashAddrID: bchchaincfg.TestNet3Params.LegacyScriptHashAddrID,
+		Bech32HRPSegwit:  "bchtest",
+		CoinbaseMaturity: 100,
+		Net:              0xf4f3e5f4,
+	})
```

Comment:
So I started switching everything to testnet 4, but quickly ran into some hurdles because we're using the bchd `chaincfg.Params` in a few places, and no such `Params` exist for testnet4. The clone params are easy and I know exactly which fields are being used there, but I can't say the same for the bchd paths, so we can't use the partial params like we do with clone params. 

Can still do it, but punting for now. We should consider opening a PR at bchd with the new parameters too.

Reply:
Sounds good.  Was really just taking notes on testnet4 here.